python/sedonadb/tests/functions/test_functions.py

```diff
@@ -2935,3 +2929,32 @@ def test_st_NRings(eng, geom, expected):
         f"SELECT ST_NRings({geom_or_null(geom)})",
         expected,
     )
+
+
+@pytest.mark.parametrize("eng", [SedonaDB, PostGIS])
+@pytest.mark.parametrize(
+    ("geom", "expected"),
+    [
+        (None, None),
+        ("POINT (1 2)", None),
```

Comment:
```suggestion
        ("POINT (1 2)", None),
        ("MULTIPOINT ((0 0), (1 1))", None),
```

I prefer to also test a non-empty version of each geometry type as well. You have all except multipoint already.